api_views/books.py

```diff
@@ -77,4 +77,31 @@ def get_by_title(book_title):
                 }
```

Comment:
The delete_book_by_title function is missing test coverage for error scenarios and successful deletion